Lib/xml/etree/__init__.py

```diff
@@ -31,3 +31,7 @@
 
 # Licensed to PSF under a Contributor Agreement.
 # See http://www.python.org/psf/license for licensing details.
+
+# Occupy all the parser files in initializer.
+
+from . import (ElementInclude, ElementPath, ElementTree, cElementTree)
```

Comment:
What happen if you use the `__all__` variable?

Reply:
if we use ```__all__``` over etree initialize then context will be little bit changed as far core behavior of this package - as all the sub-libraries(etree etc.) are already available in global name-space  by ```__all__```  in  ```__init__.py``` of xml package.  I know if we can introduce ```__all__``` from here then we can use ``` from xml.etree import *;ElementTree.somemethod``` -  Do you think to introduce this technique will be better approach comparing the current PR solution? Let me know accordingly i'll make the changes over PR